### ISSUE TITLE
Remove ts-node dep?

### ISSUE THREAD

**Issue by OmgImAlexis** [codechecks/monorepo#85]
I can't see this being called anywhere, was this meant to be added as a devDep?

https://github.com/codechecks/monorepo/search?q=ts-node

**Comment by OmgImAlexis** [comment]
This should close https://github.com/codechecks/monorepo/issues/70

**Comment by OmgImAlexis** [comment]
@krzkaczor could you confirm this. If so I'll open a PR.

**Comment by krzkaczor** [comment]
@OmgImAlexis Yes, i think you're right. `ts-node` should be a dev dependency. Please go ahead and open a PR.
